src/Inputs/InputText/InputText.stories.tsx

```diff
@@ -13,6 +13,7 @@ type Story = StoryObj<typeof InputText>
 export const Default: Story = {
   args: {
     value: 'Hello World',
+    canToggle: false,
```

Comment:
I really don't think InputText should have this toggling, it makes no sense from a components library point of view. In the areas where the FE uses <InputText /> and needs this, can it use <InputPassword /> instead?

Reply:
Only reason was because TextInput is used in secrets. I can switch their type to password, but I was not sure about this, their are technically not passwords... But sure, it is not common for standard textinput to have this option.

Output from this for me is:
**Remove visibility toggle from textinput, switch secrets input type to password.**

Reply:
<img width="1259" alt="image" src="https://github.com/ynput/ayon-react-components/assets/16327908/fb18b6d3-dc5d-434c-a83a-f8667beed021">

Reply:
Amazing, thanks! I think secrets should have probably been type=password from the beginning.